app/main/Processors/Processor.php

```diff
@@ -73,7 +74,29 @@ protected function source(string $filename): array
             return $this->source[$filename];
```

Comment:
line 61: you dont' use $locale in process method

Reply:
Yes, this is done on purpose, since in other processor this method is overridden and they need a localization parameter.

For example:

- [app/main/Processors/Statuses/Processor.php](https://github.com/andrey-helldar/Laravel-lang/blob/patch/2021-05-26/22-46/app/main/Processors/Statuses/Processor.php#L53)